services/emailService.js

```diff
@@ -0,0 +1,18 @@
+const mailgun = require("mailgun-js");
+const { MAILGUN_API_KEY, MAILGUN_DOMAIN, MAILGUN_FROM_EMAIL } = process.env;
+
+const mg = mailgun({ apiKey: MAILGUN_API_KEY, domain: MAILGUN_DOMAIN });
+
+const sendVerificationEmail = async (email, token) => {
```

Comment:
To też się powtarza, zrób sobie jedno miejsce skąd będziesz importować te funkcję i uzywać.